external/fv3fit/fv3fit/pytorch/cyclegan/modules.py

```diff
@@ -0,0 +1,235 @@
+import logging
```

Comment:
FoldTileDimension is added and the objects are refactored to work with 5-D [sample, tile, z, x, y] data (channels first, previously was channels last). Otherwise it's the same.